Corona-Warn-App/src/test/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModelTest.kt

```diff
@@ -0,0 +1,46 @@
+package de.rki.coronawarnapp.release
+
+import de.rki.coronawarnapp.main.CWASettings
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.just
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import testhelpers.TestDispatcherProvider
+import testhelpers.extensions.InstantExecutorExtension
+
+@ExtendWith(InstantExecutorExtension::class)
+class NewReleaseInfoViewModelTest {
```

Comment:
Could you add a test for what happens if there is a missmatch between titles and bodies?

Reply:
2 more tests added